### ISSUE TITLE
Finaly making it feather compatible

### ISSUE THREAD

**Issue by Gamekind135** [MeteorDevelopment/meteor-client#4228]
### Describe the feature

Hello Meteor Team and others, 

So i thoug why cant be meteor be working with feather, i understand thad meteor isnt working with optifine, but feather "removed" optifine a long time ago, Why just make a extra mod, thad idk cracks feather, i use this client since the release on so many servers, and its sad to see thad you ether use feather with some shity hacks like wurst, or you use Meteor and get one of the best clients out there, thad even supports the new versions with all features, and has an active community who help out each other, i would really apreceate it, it doesnt have to be in this verion "1.20.2", but maybe in "1.21". Im prob not the only one with thad issue, and i think thad alot of people would be grateful to have meteor working with feather. If theres a person who got it working, pls sent me a Friend request on discord "Abgehobenner". Sry thad i checked those boxes with "This feature doesn't already exist in the client. (I have checked every module and their settings on the latest dev build)" and "This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)" but i just want to know if somethinks already happening in the backround.

C ya in the responses, and if i made some mistakes while writing, pls dont hate, im a German...

### Before submitting a suggestion

- [x] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by RacoonDog** [comment]
there's no good reason to be using feather in the first place

**Comment by Big-Iron-Cheems** [comment]
We have no intention to even try and make Meteor compatible with Feather.
Their launcher is closed source, which means we couldn't even look at the code to see what changes are needed to make it work.
Also, we're definitely not going to support a launcher whose "features" are blatantly pasted from open source mods/projects.

TL;DR
Feather bad, use an actual launcher, whose job is to just launch the game, and possibly handle mods.

**Comment by Gamekind135** [comment]
but the feather client just starts fabric, and feather is just a mod, so thad means you just could copy and change it?!

**Comment by ManInMyVan** [comment]
> but the feather client just starts fabric, and feather is just a mod, so thad means you just could copy and change it?!

> closed source
